Enforced tags settings for synthetic tests (#22004)
DataDog/documentation: content/en/synthetics/settings/_index.md

Choose the default browser and device types for your [browser test][6] details.

Your options for browsers include Google Chrome, Firefox, and Microsoft Edge. Your options for devices include a large laptop, a tablet, and a small mobile device.

When you are done selecting browsers and devices, click **Save Default Browsers & Devices**.

### Default tags

Choose or add the default tags for your [API test][4], [multistep API test][5], or [browser test][6] details.

When you are done selecting related tags, click **Save Default Tags**.

### Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][11] can access the Synthetic Monitoring **Default Settings** page. To get access to the **Default Settings** page, upgrade your user to one of those two [default roles][11]. 

If you are using the [custom role feature][12], add your user to any custom role that includes `synthetics_default_settings_read` and `synthetics_default_settings_write` permissions. 

## Integration settings

{{< img src="synthetics/settings/integration_settings.png" alt="Integration Settings page" style="width:100%;">}}

### APM integration for browser tests

> Allow URLs to add APM integration headers to those URLs. Datadog's APM integration headers allow Datadog to link browser tests with APM. 

Reviewer: ```suggestion
Datadog's APM integration headers allow Datadog to link browser tests with APM. 
```

removing first sentence since it's restated with more clarity in the next line

> Define which endpoints you want to send the APM headers to by entering a URL in the **Value** field. If the endpoint is being traced and is allowed, your browser test results are automatically tied to its corresponding trace.

Reviewer: ```suggestion
Define which endpoints you want to send the APM headers to by adding a URL to the **Value** list. If the endpoint is being traced and is allowed, your browser test results are automatically tied to its corresponding trace.
```

looking at the page, it seems that `Value` isn't a field but a list, and the actual field where you enter things doesn't have a label

Use `*` to allow wider domain names. For example, adding `https://*.datadoghq.com/*` allows everything on `https://datadoghq.com/`. When you are done adding URLs, click **Save APM Integration Settings**. 

For more information, see [Connect Synthetics and APM Traces][15].

### Synthetic data collection and RUM applications

> To allow Datadog to collect RUM data from your test runs, click **Enable Synthetic RUM data collection**. If disabled, you cannot edit the RUM setting in the browser test recorder. When you are done enabling data collection, click **Save RUM Data Collection**.

Reviewer: ```suggestion
To allow Datadog to collect RUM data from your test runs, click **Enable Synthetic RUM data collection**. If disabled, you cannot edit RUM settings in the browser test recorder. To apply changes, click **Save RUM Data Collection**.
```

"When you are done enabling data collection" is misleading, I think; I would interpret it as "when you no longer want to enable data collection". I believe the intended meaning is "when you're done clicking the toggle", and have updated the copy to make that clear.

> Select a RUM application from the **Default Application** dropdown menu that collects browser test data. When you are done specifying a default application, click **Save RUM Data Applications**.

Reviewer: ```suggestion
Select a default application for new browser tests to send data. Use the **Default Application** dropdown menu to select a RUM application that collects browser test data. To apply changes, click **Save RUM Data Applications**.
```

I added a sentence for context to try to explain what "default application" means, but there may be a better way to put it.

For more information, see [Explore RUM & Session Replay][14].

### Permissions

By default, only users with the [Datadog Admin and Datadog Standard roles][11] can access the Synthetic Monitoring **Integration Settings** page. To get access to the **Integration Settings** page, upgrade your user to one of those two [default roles][11]. 

> If you are using the [custom role feature][12], add your user to any custom role that includes `synthetics_default_settings_read` and `synthetics_default_settings_write` permissions. 

Reviewer: this section is repeated
